docs/UserGuide.md

```diff
@@ -197,7 +197,7 @@ The following describes what each panel contains:
 
     * `createconsult d/10/10/2023 tt/15:00 n/John Doe n/ Foo Bar` : Creates a consultation with the students `John Doe` and `Foo Bar`.
 
-    * `createsession 3 n/John Doe` : Creates a `No. 3` Session with the student `John Doe`.
+    * `createsession s/3 n/John Doe` : Creates a `No. 3` Session with the student `John Doe`.
```

Comment:
Perhaps creates a session with session number 3 would be clearer? I think someone talked about this before

Reply:
sounds good